### ISSUE TITLE
After few hours, news pods do not get network

### ISSUE THREAD

**Issue by amapi** [projectcalico/calico#8577]
We have installed Calico with Tigera operator with WINDOWS nodes

Operator version: 1.32.5
Calico version: 3.27.2
k8s: 1.27.6

Windows node = Windows server 2022

After a few hours , if we stop a pod on windows node 

Ex: kubectl delete pod xxxx 

He stay stck on container creating 

```
Events:
  Type     Reason                  Age                 From               Message
  ----     ------                  ----                ----               -------
  Normal   Scheduled               35m                 default-scheduler  Successfully assigned mna/iis-test-egress-75d8bf86dc-qrchx to kwinlabaz0a01
  Warning  FailedCreatePodSandBox  35m                 kubelet            Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "70f665c9e5a48512dbf4629c40bfdba07689a2a2886054ea86c8b23a6b0c9e48": plugin type="calico" name="Calico" failed (add): error getting ClusterInformation: connection is unauthorized: Unauthorized
  Normal   SandboxChanged          4s (x161 over 35m)  kubelet            Pod sandbox changed, it will be killed and re-created.
```

If we delete calico windows pod, applicative pod restart ok again.
Ex: kubectl delete pod calico-node-windows-xmxrw -n calico-system

**Comment by orest-gulman** [comment]
I'm getting same error from time to time.
Calico 3.27.0
Server 2022

**Comment by caseydavenport** [comment]
I believe this is a known issue with Windows installs at the moment as discussed here: https://github.com/projectcalico/calico/issues/8379

@coutinhop just merged PRs to fix this:

- https://github.com/projectcalico/calico/pull/8563
- https://github.com/projectcalico/calico/pull/8571

Fix should be in the next v3.27 patch release, as well as v3.28.0
